_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+
+### 4. An LGPL v2.1+ package and source code
+
+This is the state which we aim the library to eventually be licensed under. Once we both obtain
+the agreement of our developers to relicense and remove [Apache 2.0][] dependencies, we will be
+able to release the MDAnalysis package under [LGPL v2.1+][].
+
+## How does this affect me?
+
+### As a contributor
+
+If you have any historical contributions you will be asked to relicense them to [LGPL v2.1+][LGPL v2.1]
+instead of the initially proposed [LGPL v3+][LGPL v3].
+
+### As a developer
+
+Ultimately we believe that the proposed change to [LGPL v2.1+][LGPL v2.1] should offer a better compatibility
+(Table 1) with whatever license you may choose to use for your own code. The interim package license changes
+may however impact you, if you believe this to be the case [please do get in touch with us](#getting-in-touch).
+
+### As a user
+
+We do not anticipate this to have a significant impact on users. There may be some slight changes to how you
+[might access MMTF files which are discussed below](#removing-apache-20-dependencies).
+
+## Next steps
+
+### Contacting contributors
+
+We will shortly be contacting every historical contributor to MDAnalysis and ask them to agree to changing
+their contribution’s license from the current “GPL v2 or any later version” to “LGPL v2.1 or any later version”.
+
+Specifically we will be asking all contributors to respond with the following statement:
+
+```
+I, <your name>, GitHub handle <your GitHub handle>, am the copyright owner of all the code I submitted
+to the MDAnalysis library using my GitHub handle. I agree to relicense all my code contributions to the
+MDAnalysis library under the LGPLv2.1+ license from this day going forward.
+
+Typed Signature
+```
+
+It is important that we hear back from as many contributors as possible. If you have contributed to
+MDAnalysis in the past but have since changed your git-linked contact details, we would kindly ask if
+you could email *licensing@mdanalysis.org* to let us know how best to contact you.
+
+### Removing Apache 2.0 dependencies
+
+As we move towards a v3.0 release of MDAnalysis, we will be looking to replace existing [Apache 2.0][]
+core dependencies. Of particular impact, this may mean that access to reading [MMTF][] files may no longer
+be available by default through the MDAnalysis library, but will instead be offered through an optionally
+installable [MDAKit](https://mdakits.mdanalysis.org/about.html).
+
+## Getting in touch
+
+We understand that license changes are a complex subject that can have a large impact on how users and
+developers may interact with the MDAnalysis library. Should you have any concerns, questions, or comments,
+we urge you to get in touch with the core developer team. You can do so over our
+[public forums[({% link index.md %}#participating) or privately by emailing *licensing@mdanalysis.org*.
+
+## Definitions
```

Comment:
Should these be moved near the top? I think it could make for a much easier reading of the blog post to clarify these points at the beginning.

Reply:
I had a comment about this elsewhere, about how we shouldn't do the "how" before we do the "why" but that a TOC might fix this. 

This is all to say that I've added a TOC.